electron/actions/app/checkForUpdates.js

```diff
@@ -4,15 +4,17 @@ import {
   shell,
   net
 } from 'electron'
-import {
-  autoUpdater
-} from 'electron-updater'
+import electronUpdater from 'electron-updater'
```

Comment:
This loads the entire electronUpdater into memory, garbage collects it, then you extract the autoupdater out and that's super inefficient. You should always import only what you need. electronUpdater, currently has no use. we only need autoupdater.

Reply:
@xyloflake https://github.com/electron-userland/electron-builder/issues/7976